rotkehlchen/icons.py

```diff
@@ -167,10 +177,8 @@ def get_icon(
         If not, all icons of the asset are queried from coingecko and cached
         locally before the requested data are returned.
         """
-        # First search custom icons
-        custom_icon_path = self.custom_iconfile_path(asset)
-        if custom_icon_path is not None:
-            return custom_icon_path
+        if self.failed_asset_ids.get(asset.identifier) is not None:
```

Comment:
you added a return value argument. Please edit the docstring to explain what this function returns.